pydis_site/apps/content/resources/guides/python-guides/discord-embed-limits.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Discord Embed Limits
+description: A guide that shows the limits of embeds in Discord and how to avoid them.
+---
+
+If you plan on using embed responses for your bot you should know the limits of the embeds on Discord or you will get `Invalid Form Body` errors:
+
+- Embed **title** is limited to **256 characters**
+- Embed **description** is limited to **4096 characters**
+- An embed can contain a maximum of **25 fields**
+- A **field name/title** is limited to **256 character** and the **value of the field** is limited to **1024 characters**
+- Embed **footer** is limited to **2048 characters**
+- Embed **author name** is limited to **256 characters**
+- The **total of characters** allowed in an embed is **6000**
+
+Now if you need to get over this limit (for example for a help command), you would need to use pagination.
+There are several ways to do that:
+
+- A library called `disputils` -> <https://pypi.org/project/disputils>
+- An experimental library made by the discord.py developer called `discord-ext-menus` -> <https://github.com/Rapptz/discord-ext-menus>
+- Make your own setup using the `wait_for()` and wait for a reaction to be added -> <https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.wait_for>
```

Comment:
This URL breaks the page on mobile (see screenshot). Not sure why it's only this link, and I don't know if this is related to anything you did in particular, but I haven't had a chance to investigate.

![image](https://user-images.githubusercontent.com/47495861/157676269-2c06cdf8-bd0a-40c2-aeb4-87f496fe526b.jpeg)

Reply:
I could use a hyperlink instead.

Reply:
I think if you user hyperlinks for these 3 bullet points we should be good to go @kkrypt0nn 

I would suggest using bolded text, and not inline code block for these links. Example: 

<img width="243" alt="image" src="https://user-images.githubusercontent.com/75038675/158081476-32895fe8-d6d9-44d0-a38d-3c375e456f8a.png">

Reply:
Sure thing, will be done next week 👍